test/e2e/reboot.go

```diff
@@ -126,8 +115,7 @@ func rebootNode(c *client.Client, provider, name string, result chan bool) {
 	}
 
 	// Get all the pods on the node.
-	podList, err := c.Pods(api.NamespaceDefault).List(
-		labels.Everything(), fields.OneTermEqualSelector(client.PodHost, name))
+	podList, err := listPods(c, api.NamespaceDefault, labels.Everything(), fields.OneTermEqualSelector(client.PodHost, name))
```

Comment:
Uhoh... Looks like this test is using the default namespace. we can fix this now or later - if later - lets create an issue before commiting so we dont forget :)